pvlib/tools.py

```diff
@@ -212,7 +212,7 @@ def _scalar_out(input):
     else:  #
         # works if it's a 1 length array and
         # will throw a ValueError otherwise
-        output = input.item()
+        output = input.values.item()
```

Comment:
this won't work if `input` is a numpy array because it doesn't have a `values` attribute. 

```python
In [3]: np.array([0]).values
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-6306ba001445> in <module>
----> 1 np.array([0]).values

AttributeError: 'numpy.ndarray' object has no attribute 'values'
```

`np.asarray` seems to work:

```python
In [9]: np.asarray(pd.Series([0])).item()
Out[9]: 0

In [10]: np.asarray(np.array([0])).item()
Out[10]: 0
```

Interesting that `np.asscalar` also emits a warning:

```python
In [7]: np.asscalar(pd.Series([0]))
/Users/holmgren/miniconda3/envs/sfacore/lib/python3.7/site-packages/numpy/lib/type_check.py:547: FutureWarning: `item` has been deprecated and will be removed in a future version
  return a.item()
Out[7]: 0
```

Reply:
Good suggestion @wholmgren. I've implemented that solution. 

Also, the Numpy docs suggest similarly under [`np.asscalar`](https://docs.scipy.org/doc/numpy-1.17.0/reference/generated/numpy.asscalar.html#numpy.asscalar).